tfs/src/main/java/hudson/plugins/tfs/model/GitCodePushedEventArgs.java

```diff
@@ -2,10 +2,11 @@
 
 import org.eclipse.jgit.transport.URIish;
 
+import java.io.Serializable;
 import java.net.URI;
 import java.net.URISyntaxException;
 
-public class GitCodePushedEventArgs {
+public class GitCodePushedEventArgs implements Serializable {

```

Comment:
Serializable is a special case. If you add this interface, all child classes must become serializable as well. And there is no guarantee they really can support it.

Fromt the **binary compatibility** standpoint it is compatible, of course